src/database/DataTypes/Item.js

```diff
@@ -283,10 +283,11 @@ export class Item extends Realm.Object {
     if (!this.isVaccine || !fromDate) return 0;
 
     const supplierCreditTransactionItems = this.transactionItems.filtered(
-      'transaction.confirmDate > $0 && transaction.type == $1 && transaction.otherParty.name == $2',
+      'transaction.confirmDate >= $0 && transaction.type == $1 &&' +
+        'transaction.otherParty.code == $2',
       fromDate,
       'supplier_credit',
-      'inventory_adjustment'
+      'invad'
```

Comment:
off-road fix - invad seems to have funny namings - more consistent to use the code in mobile